CHANGELOG.md

```diff
@@ -1,3 +1,271 @@
+# 23.03 (2022-03-31)
+
+> Checkpoint: `5323020c3f02b178f2b6807f13d89bf7f40d3cce`
+
+For the full log, check out the [git log](https://github.com/helix-editor/helix/compare/22.12..23.03).
+
+Breaking changes:
+
+- Select diagnostic range in `goto_*_diag` commands (#4713, #5164, #6193)
+- Remove jump behavior from `increment`/`decrement` (#4123, #5929)
+- Enable the Kitty keyboard protocol on terminals with support (#4939, #6170, #6194)
+- Select change range in `goto_*_change` commands (#5206)
+- Split file modification indicator from filename statusline elements (#4731, #6036)
+- Jump to symbol ranges in LSP goto commands (#5986)
+
+Features:
+
+- Dynamic workspace symbol picker (#5055)
+- Soft-wrap (#5420, #5786, #5893, #6440)
+- Initial support for LSP snippet completions (#5864, b1f7528, #6263, bbf4800, 90348b8, f87299f, #6371)
+- Add a statusline element for showing the current version control HEAD (#5682)
+- Display LSP type hints (#5934, #6312)
+
+Commands:
+
+- `:pipe-to` which pipes selections into a shell command and ignores output (#4931)
+- `merge_consecutive_selections` (`A-_`) combines all consecutive selections (#5047)
+- `rotate_view_reverse` which focuses the previous view (#5356)
+- `goto_declaration` (`gD`, requires LSP) which jumps to a symbol's declaration (#5646)
+- `file_picker_in_current_buffer_directory` (#4666)
+- `:character-info` which shows information about the character under the cursor (#4000)
+- `:toggle-option` for toggling config options at runtime (#4085)
+- `dap_restart` for restarting a debug session in DAP (#5651)
+- `:lsp-stop` to stop the language server of the current buffer (#5964)
+- `:reset-diff-change` for resetting a diff hunk to its original text (#4974)
+
+Usability improvements and fixes:
+
+- Remove empty detail section in completion menu when LSP doesn't send details (#4902)
+- Pass client information on LSP initialization (#4904)
+- Fix behavior of `auto-completion` flag for completion-on-trigger (#5042)
+- Allow specifying environment variables for language servers in language config (#4004)
+- VCS gutter: Properly handle detached git worktrees (#5097)
```

Comment:
This is actually not related to the gutter. The VCS gutter handled this case fine because we use gitoxide to load the repo.which does it's own repo.discovery. This is only related to detecting the root path (for the file picker for for example)

Reply:
Whoops, good catch!

```suggestion
- Allow detached git worktrees to be recognized as root paths (#5097)
```